.dappnode_profile

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
 export BIND_VERSION="${BIND_VERSION:-0.2.6}"
-export IPFS_VERSION="${IPFS_VERSION:-0.2.18}"
+export IPFS_VERSION="${IPFS_VERSION:-0.2.19}"
 export VPN_VERSION="${VPN_VERSION:-0.2.8}"
-export DAPPMANAGER_VERSION="${DAPPMANAGER_VERSION:-0.2.58}"
+export DAPPMANAGER_VERSION="${DAPPMANAGER_VERSION:-0.2.66}"
```

Comment:
revert personal changes 

This is/was a personal test branch and i have been testing it offline and saved my local work on my branch and pushed it and then when went back to github i found that @dappnode/core had started to move on it themselves so my last commit from my VS-Code wasnt made as a PR because I didnt know anyone was planning on implementing it just yet for everyone especially, but more importantly regarding the profile here, why is this not being updated.  I know for the pre-release theres a workflow to enter all values, but thats not for normal users, normal users follow the dev build instructions with `docker-compose build && docker-compose up` which uses these values, when the NUC12 issues first started appearing as people bought them, they kept trying to do this  and kept having to update so many core versions some major making it a process to get up to date, from their own docker generated ISOs. 
I also know things have been changing in the way a lot is organized, being built, automated QA (which is awesome,  ive found several issues so far so i wanna learn more on how its configured and i can hopefully copy all workflows for dappnode so i can replicate them in my organization without using your runners, time, and secrets etc. and dont bloat up your branches with my own tests.) but shouldn't we be keeping core DNP versions up to date across the board? not just in DNP_CORE but everywhere they're listed and used such as here?